utils/makechain.ts

```diff
@@ -2,21 +2,46 @@ import { OpenAI } from 'langchain/llms/openai';
 import { PineconeStore } from 'langchain/vectorstores/pinecone';
 import { ConversationalRetrievalQAChain } from 'langchain/chains';
 
-const CONDENSE_PROMPT = `Given the following conversation and a follow up question, rephrase the follow up question to be a standalone question.
+const PROMPT = {
```

Comment:
I noticed that PROMPT variable doesn't declare type in this script.
Here is edited result

```
const PROMPT: { [key: string]: { CONDENSE_PROMPT: string; QA_PROMPT: string; } } = {
   // ...
};
```